lib/controllers/frontend/spree/apple_pay_domain_verification_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Spree
+  class ApplePayDomainVerificationController < StoreController
+    skip_before_action :set_current_order, only: :show
+    include Spree::Core::ControllerHelpers
```

Comment:
Why do we need these lines here?

Reply:
@VygovskySergey 

`skip_before_action :set_current_order, only: :show` - we're skipping fetching/setting `spree_current_order` which is quire resources heavy, and this endpoint needs to respond quickly